docs/getting_started.md

```diff
@@ -1,64 +1,53 @@
-- [PIP](https://pip.pypa.io/en/stable/installing/)
-- [Streamlit](index.md) — This one's kind of important, so don't forget to
-  install.
+### Installing Streamlit on macOS and Linux using Pipenv
 
-If you haven't already, take a few minutes to read through [Main
-concepts](main_concepts.md) to understand Streamlit's data flow model.
+The officially-supported environment manager for Streamlit on macOS and Linux is [Pipenv](https://pypi.org/project/pipenv/):
 
-## Set up your virtual environment
+1. Navigate to your project folder: `cd <myproject>`
+2. Install Streamlit in your environment: `pipenv install streamlit`
+   <br />When you run the command above, two files named `Pipfile` and `Pipfile.lock` will appear in `myprojects/`. These files are where your Pipenv environment and its dependencies are declared.
+3. Activate your pipenv environment from the `<myproject>` folder: `pipenv shell`
+4. Test that the installation worked: `streamlit hello`
 
-Regardless of which package management tool you're using, we recommend running
-these commands in a virtual environment. This ensures that the dependencies
-pulled in for Streamlit don't impact any other Python projects
-you're working on.
+The 'Streamlit Hello' app should appear in a new tab in your web browser at [http://localhost:8501](http://localhost:8501)!
 
-- [pipenv](https://pipenv.pypa.io/en/latest/)
-- [venv](https://docs.python.org/3/library/venv.html)
-- [virtualenv](https://virtualenv.pypa.io/en/latest/)
-- [conda](https://www.anaconda.com/distribution/)
+### Installing Streamlit on Windows using Anaconda
 
-## Install Streamlit
+The officially-supported environment manager for Streamlit on Windows is [Anaconda Navigator](https://docs.anaconda.com/anaconda/navigator/). If
+you don't have Anaconda installed yet, please follow the steps provided on the [Anaconda installation page](https://docs.anaconda.com/anaconda/install/windows/).
 
-```bash
-$ pip install streamlit
-```
+Once you have installed Anaconda, please follow the steps to [set up and manage your environment](https://docs.anaconda.com/anaconda/navigator/getting-started/#managing-environments) using the Anaconda Navigator.
 
-Now run the hello world app just to make sure everything it's working:
+1. Select the "▶" icon next to your new environment. Then select "Open terminal":
+   !["Open terminal" in Anaconda Navigator](https://i.stack.imgur.com/EiiFc.png)
+2. In the terminal that appears, type: `pip install streamlit`
+3. Test that the installation worked: `streamlit hello`
 
-```bash
-$ streamlit hello
-```
+The 'Streamlit Hello' app should appear in a new tab in your web browser at [http://localhost:8501](http://localhost:8501)!
 
-## Import Streamlit
+## Part 2: Setting up a Streamlit app
 
-Now that everything's installed, let's create a new Python script and import
-Streamlit.
+Now that Streamlit is installed our virtual environment and we've validated everything is working correctly,
```

Comment:
Now that Streamlit is installed *in* our virtual environment

Reply:
Thanks